cve_bin_tool/format_checkers.py

```diff
@@ -76,9 +78,55 @@ def update_checker_table(file_path, markdown):
         f.writelines(lines)
 
 
+def insort_case_insensitive(list, x):
```

Comment:
Can this insort algorithm be replaced by [bisect](https://docs.python.org/3.7/library/bisect.html#)'s implementation?

Reply:
On 2nd thought do we really need insort in here? If you're trying to do case insensitive sorting we could use inbuilt methods to achieve the same right?

Reply:
I couldn't find a bisect module function to do case insensitive insort
And we need insort because I'm not inserting an element and then sorting the entire array (O(n log n))
I am searching for the index and then inserting the element at required index (O(n))

Reply:
IMO readability holds priority than performance, especially since we are dealing with a few hundred lines. The performance difference between the 2 approaches would only be prominent when we are dealing with 10k+ lines.

Reply:
Makes sense. I'll push the changes. Thankyou